### PR TITLE
Submitting RiotGear to the CDN

### PR DESCRIPTION
**Pull request by gregorypratt**

### CODE REVIEW

File: files/riotgear/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "riotgear",
+  "repo": "RiotGear/rg",
+  "files": {
+    "basePath": "dist/"
```
Comment (jimaek) [inline files/riotgear/update.json]: Can you add an `include` parameter and include only the min file? The unminified file is not required in production

**Comment by gregorypratt** [comment]
Updated to include an include parameter

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Thank you

**Pull request closed without merging**